not-supported/system-restore.sh

```diff
@@ -64,7 +64,7 @@ Please do that before you can view backups."
     exit 1
 fi
 # Get needed variables
-ENCRYPTION_KEY="$(grep "ENCRYPTION_KEY=" "$DAILY_BACKUP_FILE" | sed 's|.*ENCRYPTION_KEY="||;s|"||')"
+ENCRYPTION_KEY="$(grep "ENCRYPTION_KEY=" "$DAILY_BACKUP_FILE" | sed 's|.*ENCRYPTION_KEY=\x27||;s|\x27||')"
```

Comment:
```suggestion
ENCRYPTION_KEY="$(grep "ENCRYPTION_KEY=" "$DAILY_BACKUP_FILE" | sed 's|.*ENCRYPTION_KEY=||;s|\'||g;s|"||g')"
```

Reply:
This pull request should not be considered as "this is how the problem should be solved" PR. I know far too little about sed to say what is the best solution here and it would take me far too much time to figure it out. I just tried my solution and it worked. Do whatever you think is best. I probably should have just reported this as an issue instead of a PR.

Reply:
all right. just tested and my proposal seems to work

Reply:
```
export TEST='ENCRYPTION_KEY="dfjio'dkfjl'"'
echo "$(echo "$TEST" | sed "s|.*ENCRYPTION_KEY=||;s|'||g;s|\"||g")"
dfjiodkfjl
```